daemon/state.go

```diff
@@ -154,6 +159,7 @@ func (s *State) IsRunning() bool {
 	return res
 }
 
+// GetPid holds the process id of a container (perhaps this should be moved up into CommonContainer
```

Comment:
Not indicated by golint, but it is in line with other changes that golint does suggest.